lib/types/autocrats.ts

```diff
@@ -24,17 +24,13 @@ export type AutocratProgram =
 /**
  * DAOs
  */
-export type DaoAccountV0_1 = MergeWithOptionalFields<
-  IdlAccounts<AutocratV0>["dao"],
-  IdlAccounts<AutocratV0_1>["dao"]
->;
-export type DaoAccountV0_2 = MergeWithOptionalFields<
-  DaoAccountV0_1,
-  IdlAccounts<AutocratV0_2>["dao"]
->;
 export type DaoAccount = MergeWithOptionalFields<
-  DaoAccountV0_2,
-  IdlAccounts<AutocratV0_3>["dao"]
```

Comment:
Just a question for me.. What is the difference between the merge with optional or just adding them as | | |?

Reply:
My goal was to have a structure where a field that is not present in ALL versions of the program is marked as optional. Using `|` takes all fields without taking care of  whether the field is always available.

But actually, looking closer into what I did, it was not working as intended so I reverted to the old version that supposes that the latest version is used and marks fields only present in other versions as optional